Updated rkt demo and docs

gunjan5 opened the pull request.

djosborne commented:
@gunjan5 can you modify this so there's no file location movement yet? The diff is difficult to make sense of atm. Can do a follow up PR to toy with structure.

gunjan5 commented:
@djosborne there's more than location movement. Location movement still shows up as normal diff, it's just the file I modified and added some more content and then broke it down to two files `/vagrant-coreos/index` and `/tutorials/basic`. I'm creating a branch with no location change

gunjan5 commented:
@djosborne toy PR to see the diff view https://github.com/gunjan5/calico/pull/1/files

djosborne commented:
I believe this Fixes #26 and Fixes #229

robbrockbank commented on master/getting-started/rkt/tutorials/basic.md:
````diff
@@ -0,0 +1,196 @@
+--set-env=IP=autodetect --net=host quay.io/calico/node:latest &
+```
+
+This will create a rkt container called `calico-node`.
````
Is it really called calico-node?

gunjan5 replied:
hmm not really, I'll change it

caseydavenport commented on _data/master/navbars/getting-started.yml:
```diff
@@ -145,10 +145,12 @@ toc:
     section:
-    - title: Vagrant
-      path: /getting-started/rkt/vagrant/
+    - title: 'Vagrant/VirtualBox: CoreOS'
```
Why the single quotes? I think we don't want that.

robbrockbank replied:
That might be required given the string has a colon in it...

robbrockbank replied:
Could just change to Vagrant/VirtualBox/CoreOS    :-)

robbrockbank replied:
without the smiley face.

caseydavenport commented on master/getting-started/rkt/installation/cloud-config/other-node-config.yaml:
```diff
@@ -2,9 +2,9 @@
 coreos:
   update:
-    reboot-strategy: off
+    reboot-strategy: 'off'
```
I don't think this is necessary, is it?

robbrockbank commented:
This was checked in under another PR... Closing.
